packages/web/src/scss/components/UNSTABLE_ProductLogo/README.md

```diff
@@ -0,0 +1,30 @@
+# UNSTABLE ProductLogo
+
+⚠️ This component is UNSTABLE. It may significantly change at any point in the future. Please use it with caution.
```

Comment:
Just [formatting nitpicking](https://github.com/lmc-eu/spirit-design-system/pull/1533/files#diff-a54c1ab3e7573039702748db2be76867a09202de0562599de3c7b9bcbeeb1035) as in `PartnerLogo` – feel free to skip them or implement them in both PRs.